Source/Core/VideoCommon/GXPipelineTypes.h

```diff
@@ -19,7 +19,7 @@ namespace VideoCommon
 // As pipelines encompass both shader UIDs and render states, changes to either of these should
 // also increment the pipeline UID version. Incrementing the UID version will cause all UID
 // caches to be invalidated.
-constexpr u32 GX_PIPELINE_UID_VERSION = 6;  // Last changed in PR 10890
+constexpr u32 GX_PIPELINE_UID_VERSION = 7;  // Last changed in PR 11859
```

Comment:
Technically, adding a new field to an existing bitfield where zero is the same as the current usage *shouldn't* break any UIDs